src/cgame/cg_main.c

```diff
@@ -821,25 +821,54 @@ void CG_RestoreProfile(void)
 }
 
 /**
- * @brief Try to exec a cfg file if it is found
+ * @brief Checks whether a cfg file exists or not
  * @param[in] filename
- * @return
+ * @return qtrue if file exists
  */
-qboolean CG_execFile(const char *filename)
+qboolean CG_ConfigFileExists(const char *filename)
 {
 	int handle = trap_PC_LoadSource(va("%s.cfg", filename));
-
 	trap_PC_FreeSource(handle);
+	return handle ? qtrue : qfalse;
+}
 
-	if (!handle)
+/**
+ * @brief Execs a .cfg file
+ * @param[in] filename
+ */
+void CG_execFile(const char *filename)
+{
+	trap_SendConsoleCommand(va("exec %s.cfg\n", filename));
+}
+
+/**
+ * @brief Executes map-specific autoexec files
+ * @param[in] cheats
+ */
+void CG_MapAutoexec(qboolean cheats)
```

Comment:
nit: consider renaming `cheats` arg to `useDefaults` or similar - 2nd callsite passes `qfalse` despite `cgs.sv_cheats == 1`